src/flamenco/runtime/fd_acc_mgr.c

```diff
@@ -317,7 +317,12 @@ fd_acc_mgr_save_non_tpool( fd_acc_mgr_t *          acc_mgr,
   if( fd_funk_val_truncate( account->rec, reclen, fd_funk_alloc( acc_mgr->funk, wksp ), wksp, &err ) == NULL ) {
     FD_LOG_ERR(( "unable to allocate account value, err %d", err ));
   }
-  return fd_acc_mgr_save( acc_mgr, account );
+  err = fd_acc_mgr_save( acc_mgr, account );
+  fd_acc_saved_fun saved_fun = acc_mgr->saved_fun;
+  if( saved_fun != NULL ) {
+    (*saved_fun)( account->pubkey, txn, acc_mgr->saved_fun_arg );
+  }
+  return err;
 }
```

Comment:
why a callback? this seems like a bad pattern when the replay tile has all of the info already to push these things out

Reply:
i don't think the replay tiles knows which accounts were updated